### DISCUSSION TITLE
"dragonEggHandDropping"

### DISCUSSION THREAD

**Discussion by ghost** [Minecraft-AMS/Carpet-AMS-Addition#68] [Ideas]
A simple click drops the Dragon Egg, instead of an random position teleport.

**Comment by 1024-byteeeee** [comment]
> 只需点击一下就会掉落龙蛋，而不是随机位置的传送。

### Changelog
#### https://minecraft-ams.github.io/carpetamsaddition/2024/01/01/v2.23.0/

**Reply by ghost** [reply]
Great, thank you 😊

**Reply by 1024-byteeeee** [reply]
My pleasure
